src/PhpPact/Standalone/MockService/MockServer.php

```diff
@@ -12,34 +13,19 @@
 
 /**
  * Ruby Standalone Mock Server Wrapper
- * Class MockServer.
  */
 class MockServer
 {
-    /** @var MockServerConfig */
-    private $config;
+    private MockServerConfig $config;
 
-    /** @var MockServerHttpService */
-    private $httpService;
+    private MockServerHttpService $httpService;
 
-    /** @var ProcessRunner */
-    private $processRunner;
+    private ProcessRunner $processRunner;
 
-    /**
-     * MockServer constructor.
-     *
-     * @param MockServerConfig           $config
-     * @param null|MockServerHttpService $httpService
-     */
     public function __construct(MockServerConfig $config, MockServerHttpService $httpService = null)
     {
-        $this->config         = $config;
-
-        if (!$httpService) {
-            $this->httpService = new MockServerHttpService(new GuzzleClient(), $this->config);
-        } else {
-            $this->httpService = $httpService;
-        }
+        $this->config = $config;
+        $this->httpService = $httpService ?: new MockServerHttpService(new GuzzleClient(), $this->config);
```

Comment:
> Even though `?:` (Ternary Operator) and `??` (Null Coalescing Operator) give the same result, I personally think `??` is a bit better

I'm happy with either but note that it is also changed here too

https://github.com/pact-foundation/pact-php/pull/290/files#diff-bcaf7e4069d08d80a198074bbb5ed5598ca0686e450f03ae4d0e5e64797a1cc3R15

I am going to merge this PR to help move things along, I've had a quick scan and it looks sensible, I'm going through all the PR's at the moment because I've got PHP fever :) Looking forward to going over your FFI ones 👍🏾